packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -230,20 +256,7 @@ function useMediaUploadQueue() {
           let newFile = file;
           let newPosterFile = posterFile;
 
-          if (
-            resource.type === 'video' &&
-            resource.isMuted !== null &&
-            additionalData?.web_stories_is_muted === undefined
-          ) {
-            additionalData.web_stories_is_muted = resource.isMuted;
-          }
-
-          if (resource?.baseColor) {
-            additionalData.meta = {
-              ...additionalData.meta,
-              web_stories_base_color: resource.baseColor,
-            };
-          }
+          const additionalData = addAdditionalData(_additionalData, resource);
```

Comment:
Needed as function was too complex.

Reply:
That doesn't feel very readable. I think there's a bigger opportunity to reduce complexity in this function more holistically.

Let's do `// eslint-disable-next-line complexity` for now and solve complexity in a separate ticket.